### ISSUE TITLE
What's the difference between .ddev/.homeadditions and .ddev/homeadditions?

### ISSUE THREAD

**Issue by ahillio** [drud/ddev#2767]
I'm having the darndest time getting my shell/vim config into my containers... and one specific question I have is in my project, what's the difference between `.ddev/homeadditions` and `ddev/.homeadditions`?  I have near-identical directories, one hidden the other not.  `.ddev/homeadditions` (the visible dir) seems to contain the config that gets copied to container.  What's the point of the identical hidden directory?

**Comment by rfay** [comment]
There are two ways to do homeadditions, global (~/.ddev/homeadditions - the homeadditions in your global .ddev directory in your home directory) and per-project (.ddev/homeadditions, in the .ddev directory of your project.) When you put something in either place, it gets copied into the web container's home directory when it comes up. 

The actual mechanism that gets used is that at start time, the contents of the global homeadditions gets copied into the project .homeadditions, which is hidden because you never need to know that it exists. But if you look inside it you'll see what is copied from your global .ddev/homeadditions. The bottom line: Ignore directories with a dot in front of them like this one. 

The docs are at https://ddev.readthedocs.io/en/stable/users/extend/in-container-configuration/#in-container-home-directory-and-shell-configuration - let me know if you think they can be improved.

**Comment by ahillio** [comment]
Thanks Randy.  I was amazed to see your response last night.  When I posted thought I thought "gee, I hope Randy doesn't see this until going into work tomorrow" (as in I hope you're enjoying the weekend and not helping strangers use your free software) and then probably within the hour you answered my question.  If there was a chart ranking best-supported open source projects Ddev would be at the top!

I added a line that clarifies what I was wondering.  No pressure to merge it, but figured I'd offer it.  Thanks again.
